Integrations/integration-fireeye.yml

```diff
@@ -0,0 +1,274 @@
+        }
+        return res;
+    }
+
+    var token = sendRequest(commandDictionary.login.url, commandDictionary.login.method).Headers['X-Feapi-Token'];
+    var result;
+    switch (command) {
+        case 'test-module':
+            sendRequest(commandDictionary.logout.url, commandDictionary.logout.method, token);
+            return 'ok';
+        case 'fe-report':
+            var response = sendRequest(commandDictionary[command].url, commandDictionary[command].method, token, commandDictionary[command].setContentType, args);
+            var filename = getFileName(args);
+            result = {Type: 9, FileID: saveFile(response.Body), File: filename, Contents: filename};
+            break;
+        case 'fe-submit':
+            if (args.profiles) {
+                args.profiles = args.profiles.split(',');
+                for (var k in args.profiles) {
+                  args.profiles[k] = args.profiles[k].trim();
+                }
+            }
+            id = args.upload || args.uploadFile;
+            delete(args.upload)
+            delete(args.uploadFile)
+            res = httpMultipart(
+                 server + 'submissions',
+                 id,
+                 {
+                     Headers: {
+                        'X-FeClient-Token': params.clientToken ? [params.clientToken] : undefined,
+                        'X-FeApi-Token': token,
+                        'Accept': ['*/*']
+                     },
+                 },
+                 {
+                    'options': JSON.stringify(args)
+                 },
+                 params.insecure,
+                 params.proxy,
+                 undefined,
+                 'filename'
+             );
+             if (res.StatusCode < 200 || res.StatusCode >= 300) {
+               sendRequest(commandDictionary.logout.url, commandDictionary.logout.method, token);
+               throw 'Request submissions failed, request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+              }
+              return res.Body;
+        default:
+            respoonse = sendRequest(replaceInTemplatesAndRemove(commandDictionary[command].url, args), commandDictionary[command].method, token, commandDictionary[command].setContentType, args);
+            result = respoonse.Body;
+            contentType = respoonse.Headers && respoonse.Headers['Content-Type'] && respoonse.Headers['Content-Type'][0];
+            if (contentType && contentType.indexOf('application/json') !== -1) {
+              result = JSON.parse(respoonse.Body);
+            }
+            if (contentType && contentType.indexOf('application/xml') !== -1) {
+              result = JSON.parse(x2j(respoonse.Body));
+            }
+    }
+    sendRequest(commandDictionary.logout.url, commandDictionary.logout.method, token);
+    return result;
+  type: javascript
+  commands:
+  - name: fe-report
+    arguments:
```

Comment:
I didn't add anything to the context yet. I want to verify this is working and replacing the go module first of all..